### PR TITLE
more usable space for the classic JSON edit overlay

### PR DESCRIPTION
**Pull request by ArnoldSmith86**
Just improved the layout a bit. Feedback is welcome.

### CODE REVIEW

**Comment by ArnoldSmith86** [comment]
PR-SERVER-BOT: You can play around with it here: http://212.47.248.129:3918/pr-test (or any other room on that server)

File: client/css/editmode.css [hunk 1]
```diff
@@ -30,9 +30,16 @@
...
+}
+
+#editJSONoverlay h1 {
+  margin: 0;
+}
+
 #editJSONoverlay textarea {
-    height: 100%;
-    min-height: 100%;
+  height: 100%;
```
Comment (RaphaelAlvez) [inline client/css/editmode.css:42]: .....

we reintroduced this yesterday 😑

we need it for mobile. I'll take a look at this PR

**Review by RaphaelAlvez — CHANGES_REQUESTED** [review]

**Review by rogerl50 — APPROVED** [review]
Code looks good, basic testing seems to work. Nice plug for the JSON editor.

File: client/css/editmode.css (hunk 1 above)
Comment (ArnoldSmith86) [reply client/css/editmode.css:42]: Did you test it? I thought it might be enough without it now.

Comment (ArnoldSmith86) [reply client/css/editmode.css:42]: I just tested on my phone and I think it's good enough. The problem with the `min-height` is that the buttons are off screen and if the text is long enough you get two nested scrollbars. So if it's good enough without it, I would prefer that.

**Comment by ArnoldSmith86** [comment]
I really don't see the `min-height` as an improvement.

File: client/css/editmode.css (hunk 1 above)
Comment (RaphaelAlvez) [reply client/css/editmode.css:42]: Before 

After

Comment (ArnoldSmith86) [reply client/css/editmode.css:42]: Holy shit, are you blind? :smile:

We should at least only do that when the viewport isn't high enough. It feels so broken to have to scroll to the buttons on a 4k screen.

**Comment by RaphaelAlvez** [comment]
500px seems to be a good limit based on the screen resolutions to chrome shows in dev tools. tablets should not trigger it neither portrait mode.

**Comment by RaphaelAlvez** [comment]
@rogerl50 I added some code. mind reviewing it again?

**Comment by rogerl50** [comment]
> @rogerl50 I added some code. mind reviewing it again?

Sorry, you'll need someone who actually knows css. I just copy from other things when I write css :smiley:

**Comment by ArnoldSmith86** [comment]
He meant just redo whatever you did for your last review because stuff changed.

**Review by rogerl50 — APPROVED** [review]
Can't comment on the @media addition, but other changes still look good.

**Pull request merged**